external/fv3fit/tests/emulation/layers/test_architectures.py

```diff
@@ -91,11 +102,33 @@ def test_no_weight_sharing_num_weights():
     assert num_weights_expected == total
 
 
-@pytest.mark.parametrize("layer_cls", [CombineInputs, MLPBlock, RNNBlock])
+@pytest.mark.parametrize("layer_cls", [CombineInputs, MLPBlock, HybridRNN, RNN])
 def test_from_config(layer_cls):
 
     layer = layer_cls()
     config = layer.get_config()
     rebuilt = layer_cls.from_config(config)
 
     assert isinstance(rebuilt, layer_cls)
+
+
+@pytest.mark.parametrize(
+    "connector_cls, hidden_out",
+    [
+        (StandardOutputConnector, _get_tensor((10, 64))),
+        (RNNOutputConnector, _get_tensor((10, 79, 64))),
+    ],
+)
+def test_OutputConnectors(connector_cls, hidden_out):
```

Comment:
Retarget to tests of `RNN` and `HybridRNN` please if these coverage isn't already there.

Reply:
`test_ArchitectureConfig` adds coverage to whole "architecture".